tests/test_pipeline_generation.py

```diff
@@ -21,3 +21,11 @@ def test_generate_pipeline(pipeline):
             in ephys_report.UnitLevelReport.parents(),
         ]
     )
+    
+    # test the connection between quality metric tables
+    assert all(
+        [
+            ephys.QualityMetrics.full_table_name in ephys_report.QualityMetricSet.parents(),
+            ephys_report.QualityMetricSet.full_table_name in ephys_report.QualityMetricReport.parents()
+        ]
+    )
```

Comment:
```suggestion
    assert ephys.QualityMetrics.full_table_name in ephys_report.QualityMetricSet.parents()
```

Reply:
Thanks @JaerongA.  I don't think that the second assertion is required since both tables are defined in the same schema (i.e. `ephys_report`).